packages/core/src/lint.ts

```diff
@@ -105,7 +105,10 @@ export async function lintDocument(opts: {
     resolvedRefMap,
     ctx,
   });
-  return ctx.problems.map((problem) => config.addProblemToIgnore(problem));
+  return {
```

Comment:
This change is needed in order to detect specVersion for proper validation message replacing `API` with `Test` as message subject.
But will affect dependent packages:
`const { problems: results } = await lintDocument({`
 instead of
`const result = await lintDocument({`


@RomanHotsiy / @tatomyr will it be ok to apply this ?
Alternative solution can be read document one more time to detect specVersion, or change the final message not to have specific subject information, like `API`